### PR TITLE
Proper highlighting of library items

### LINKED ISSUE #999: Not the whole line in library view is highlighted
In library view, when mouse is over a book, we grey the name/date/size/content type columns.
But we should also grey the icon and the button open/download.

### PR DESCRIPTION
**Pull request by veloman-yunkan**
Fixes #999

### CODE REVIEW

**Comment by veloman-yunkan** [comment]
Note that there is a slight change in the position of the thumbnails (but I bet that you wouldn't have noticed it without this hint).

File: src/contentmanagermodel.cpp [hunk 1]
```diff
@@ -24,27 +24,62 @@ int ContentManagerModel::columnCount(const QModelIndex &parent) const
...
-            return r;
+            return getIcon(r.toByteArray());
 
         const QString faviconUrl = r.toString();
         if ( !faviconUrl.isEmpty() )
             td.addDownload(faviconUrl, item->getBookId());
+        return QVariant();
+    } else {
+        return r;
     }
```
[the remark this answers is not in the document]
Comment (veloman-yunkan) [reply src/contentmanagermodel.cpp]: Done

**Comment by veloman-yunkan** [comment]
> It seems the separator lines doesn't go under the open/download button.
> 
> On Fedora, I have a strange "widget" selected between the first icon and the name. Don't know what it is exactly. Pressing tab deselect it (and so "hide" it)

Fixed those and a couple of other issues too.

**Review by kelson42 — APPROVED** [review]
LGTM

**Comment by kelson42** [comment]
@veloman-yunkan Let you reorganize your commit and merge. Thx!

**Comment by veloman-yunkan** [comment]
> @veloman-yunkan Let you reorganize your commits, rebase and merge. Thx!

Shan't we wait for @mgautierfr's feedback too?

**Comment by kelson42** [comment]
@veloman-yunkan I think we can go as he already has a first look to the code and didn‘t find anything big.

**Comment by veloman-yunkan** [comment]
> @veloman-yunkan I think we can go as he already has a first look to the code and didn‘t find anything big.

@kelson42 This PR is now mergeable

**Pull request merged**